src/Form/CitizenProjectCommandType.php

```diff
@@ -94,6 +94,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $citizenProject = $builder->getData()->getCitizenProject();
 
+        if (!$citizenProject instanceof CitizenProject) {
```

Comment:
Can you place this logic in the listener on the `preSetData`? I think you can also place the `committees` and` committees_search` fields logic.